stl/src/tzdb.cpp

```diff
@@ -182,6 +182,20 @@ namespace {
         return _Fun(en, resultLength, status);
     }
 
+    template <>
+    struct _Tzdb_deleter<UEnumeration> {
+        void operator()(UEnumeration* _En) const noexcept {
+            __icu_uenum_close(_En);
+        }
+    };
+
+    template <>
+    struct _Tzdb_deleter<UCalendar> {
+        void operator()(UCalendar* _Cal) const noexcept {
+            __icu_ucal_close(_Cal);
+        }
+    };

```

Comment:
If `_Tzdb_deleter` is moved into `std`, I believe these explicit specializations will need to be `_STD` qualified (but we don't need to move them into a `_STD_BEGIN` block, I'm pretty sure), ditto for the uses below.

Reply:
(Actually it's an error, will need a larger change)